examples/ensemble/plot_gradient_boosting_quantile.py

```diff
@@ -32,41 +36,87 @@ def f(x):
 y += noise
 y = y.astype(np.float32)
 
+# Split into train, test datasets.
+X_train, X_test, y_train, y_test = train_test_split(X, y)
+
 # Mesh the input space for evaluations of the real function, the prediction and
-# its MSE
+# its MSE.
 xx = np.atleast_2d(np.linspace(0, 10, 1000)).T
 xx = xx.astype(np.float32)
 
-alpha = 0.95
-
-clf = GradientBoostingRegressor(loss='quantile', alpha=alpha,
-                                n_estimators=250, max_depth=3,
-                                learning_rate=.1, min_samples_leaf=9,
-                                min_samples_split=9)
-
-clf.fit(X, y)
-
-# Make the prediction on the meshed x-axis
-y_upper = clf.predict(xx)
-
-clf.set_params(alpha=1.0 - alpha)
-clf.fit(X, y)
-
-# Make the prediction on the meshed x-axis
-y_lower = clf.predict(xx)
-
-clf.set_params(loss='ls')
-clf.fit(X, y)
-
-# Make the prediction on the meshed x-axis
-y_pred = clf.predict(xx)
+# %%
+# Let's fit a model based trained with MSE loss.
+gbr_mse = GradientBoostingRegressor(loss='ls', n_estimators=250, max_depth=3,
```

Comment:
I wonder if reducing the number of estimators to 100 would not make the model overfit less. But maybe this is not a big deal as long as the comments explains what's happening.